src/content/en/updates/2016/07/capture-stream.md

```diff
@@ -0,0 +1,104 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The captureStream() method enables a MediaStream to be captured from a canvas, audio or video element, on Android and desktop.
+
+{# wf_updated_on: 2016-10-20 #}
+{# wf_published_on: 2016-10-20 #}
+{# wf_tags: canvas,chrome53,gif,media,security,webrtc #}
+{# wf_featured_image: /web/updates/images/2016/07/autoplay/featured.jpg #}
+
+<style>
+video {
+  border: 1px solid #444;
+  max-width: 80%;
+}
+@media max-width: 500px {
+  video {
+    max-width: 100%;
+  }
+}
+</style>
+
+# Capture a media stream from a canvas, video or audio element {: .page-title }
+
+{% include "web/_shared/contributors/samdutton.html" %}
+
+<p class="intro">The <a href="https://w3c.github.io/mediacapture-fromelement/#dfn-capturestream" title="W3C captureStream() spec"><code>captureStream()</code></a> method makes it possible to capture a `MediaStream` from a <code>&lt;canvas&gt;</code>, <code>&lt;audio&gt;</code> or <code>&lt;video&gt;</code> element.</p>
```

Comment:
We no longer need/use the intro class, please remove. If you want this bit to be on the index page, use the `{# wf_featured_snippet #} tag